x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -184,7 +180,7 @@ async function get(
         if (connector != null) {
           try {
             mappings = await casesClientInternal.configuration.getMappings({
-              connector: transformESConnectorToCaseConnector(connector),
```

Comment:
We no longer need to do the transformation here because that functionality/responsibility has been moved to the service layer.